packages/thea/src/config/evm/assets.ts

```diff
@@ -0,0 +1,17 @@
+import { Asset } from "../types";
+
+const ETH: Asset = {
+  id: "123",
```

Comment:
the ids here are placholders, please check with @zktony to get the real values here so it can be used.

Reply:
But these Id's on sepolia network. not on polkadex network. Are these not correct ?

Reply:
@nuel77

Reply:
no, id is for polkadex. sepholia only has a contract address for the tokens.

Reply:
But we did not added it yet. We will add it later. This is not being used for Polkadex. It's just for sepolia. Basically, when we will add the code for THEA withdrawal, then we will add it. Currently, This PR has the code for deposit only.

1. Sepolia to Polkadex 
2. AssetHub to Polkadex

Reply:
yes but the id given here is wrong.

Reply:
@nuel77  I put correct id's now.